.github/workflows/cts_ci.yml

```
@@ -24,10 +25,27 @@ jobs:
           submodules: true
       - name: Configure CMake
         working-directory: ${{ env.container-workspace }}
-        run: bash /scripts/configure.sh
+        run: |
+          bash /scripts/configure.sh \
+            -DSYCL_CTS_EXCLUDE_TEST_CATEGORIES="${{ env.container-workspace }}/ci/${{ matrix.sycl-impl }}.filter" \
+            -DSYCL_CTS_MEASURE_BUILD_TIMES=ON
       - name: Build 'oclmath'
         working-directory: ${{ env.container-workspace }}/build
         run: cmake --build . --target oclmath
       - name: Build 'util'
         working-directory: ${{ env.container-workspace }}/build
         run: cmake --build . --target util
+      - name: Build all supported test categories
+        working-directory: ${{ env.container-workspace }}/build
+        run: |
+          TS_BEFORE=$(date +%s)
+          cmake --build . --target test_all -- -j${{ env.parallel-build-jobs }}
```

While you are here :-)
```suggestion
          cmake --build . --target test_all -- --parallel  ${{ env.parallel-build-jobs }}
```
I am unsure about the `--` in the middle. It means that you push the option to be handled by the underlying build tool textually.
If you do not have `--`, this is actually an option understood by CMake which will be translated in the right syntax for the underlying tool.
https://cmake.org/cmake/help/latest/manual/cmake.1.html#build-a-project

---

You are getting quite pedantic on me here... Fixed

---

I noticed with you some sense of deutsche Qualität so I take advantage of it. ;-)